Changelog.md

```diff
@@ -3,6 +3,7 @@
 ## 0.0.34
 
 - Provide hover for Kamelet properties based on embedded snapshot Kamelet catalog version
+- Update default Camel Catalog from 3.9.0 to 3.10.0

```

Comment:
Might be made stronger by adding "version" -- i.e. "Update default Camel Catalog version from 3.9.0 to 3.10.0"